src/pycares/__init__.py

```diff
@@ -504,11 +502,12 @@ def __init__(self,
         if local_dev:
             self.set_local_dev(local_dev)
 
+        # Ensure the shutdown thread is started
```

Comment:
Rather than swallow the error in case we try to cleanup on interpreter shutdown, I chose to start the thread as soon as the channel is ready, since we are going to need it anyway.